models/graphql/intermediate/base/int_shopify_gql__order_line.sql

```diff
@@ -0,0 +1,52 @@
+{{ config(enabled=var('shopify_api', 'rest') == var('shopify_api_override','graphql')) }}
+
+with order_line as (
+    
+    select *
+    from {{ var('shopify_gql_order_line') }}
+),
+
+tax_line as (
+
+    select *
+    from {{ var('shopify_gql_tax_line') }}
+),
+
+tax_line_aggregated as (
+
+    select
+        tax_line.order_line_id,
+        tax_line.source_relation,
+        sum(tax_line.price_shop_amount) as order_line_tax
+    from tax_line
+    group by 1,2
+),
+
+fulfillment_order_line_item as (
```

Comment:
This model can be disabled via the `shopify_gql_using_fulfillment_order_line_item` variable. We need to add a conditional block for that here.

Reply:
Adding